src/main/java/mate/academy/internetshop/model/Bucket.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.internetshop.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Bucket {
+    private static Long idGenerator = 0L;
```

Comment:
optional: we can create separate class IdGenerator and create methods here which will increment id for all models (getItemId(), getBucketId()...)